src/main/java/io/nats/service/ServiceUtil.java

```diff
@@ -0,0 +1,29 @@
+package io.nats.service;
+
+import java.time.Duration;
+
+import static io.nats.client.support.Validator.nullOrEmpty;
+
+public class ServiceUtil {
+
+    public static final String PING = "PING";
+    public static final String INFO = "INFO";
+    public static final String SCHEMA = "SCHEMA";
+    public static final String STATS = "STATS";
+    public static final String DEFAULT_SERVICE_PREFIX = "$SRV.";
+    public static final String QGROUP = "q";
+
+    public static final Duration DEFAULT_DRAIN_TIMEOUT = Duration.ofSeconds(5);
+    public static final long DEFAULT_DISCOVERY_MAX_TIME_MILLIS = 5000;
+    public static final int DEFAULT_DISCOVERY_MAX_RESULTS = 10;
+
```

Comment:
`baseSubject` is your `action` above, why not call it the same

Reply:
Also with the exception of the prefix - if one is given to straddle account boundaries, all other tokens (verb, name, id) should be uppercase.

Reply:
Yeah, prefix replacement is unresolved so I wouldn't worry about his for right now. It will be addressed when the decision about how to modify the prefix for across account is made. Will it be done KV/Object Store style or will it be done JS Prefix style?